ports/stm32/eth_phy.h

```diff
@@ -56,6 +56,14 @@
 #define PHY_SPEED_100FULL  (6)
 #define PHY_DUPLEX         (4)
 
+// PHY interrupt registers (common for LAN87xx and DP838xx)
```

Comment:
These constants aren't used anywhere.

Reply:
Ah yes I'd initially thought I'd be able to use phy interrupts to trigger the link up/down events, however in many/most cases (including the H5 nucleo) the phy used shares a pin for both ref clock out and interrupt out aka if you're using the reference clock (like we are) you can't get interrupts from the phy.